src/gwt/src/org/rstudio/studio/client/workbench/commands/CmdConstants.java

```diff
@@ -3210,6 +3210,22 @@ public interface CmdConstants extends Constants {
     @DefaultStringValue("Install the package and restart R") // $NON-NLS-1$
     String buildAllDesc();
     
+    // buildIncremental
+    @DefaultStringValue("Install Package incrementally") // $NON-NLS-1$
```

Comment:
Should we update these values as well, to reflect the new defaults in the `_en` file?